draft-huitema-dprive-dnsoquic.md

```diff
@@ -152,14 +165,14 @@ No attempt is made to address the recursive to authoritative scenarios.
 Authoritative resolvers are discovered dynamically through NS records. It is
 noted that at the time of writing work is ongoing in the DPRIVE working group to
 attempt to address the analogous problem for DNS/TLS
-[@?I-D.bortzmeyer-dprive-resolver-to-auth]. In the absence of an agreed way for
+{{?I-D.bortzmeyer-dprive-resolver-to-auth}}. In the absence of an agreed way for
```

Comment:
This draft expired over a year ago... I think referencing  draft-ietf-dprive-phase2-requirements would be better as that is adopted?

Reply:
OK.